datasets/gsm-plus/prepare.py

```diff
@@ -48,16 +53,23 @@
             "reversing_operation",
         ],
     )
+    parser.add_argument("--add_rounding_instructions", type=bool, default=True)
```

Comment:
I think if you specify it in this way, it will be impossible to disable? E.g. how do you run this script without that argument set to true